### DISCUSSION TITLE
Legacy version on regions plugin?

### DISCUSSION THREAD

**Discussion by anttieintola** [katspaugh/wavesurfer.js#2830] [Q&A] [answered]
Is there still an older version available of regions plugin as a CDN?

Can't seem to find one and the one I got from [here](https://www.unpkg.com/wavesurfer@1.3.4/dist/plugin/wavesurfer.regions.min.js")  gives an `wavesurfer.regions.min.js:3 Uncaught TypeError: a.util.extend is not a function` error

**Comment by katspaugh** [comment, marked as answer]
https://www.unpkg.com/wavesurfer.js@6/dist/plugin/wavesurfer.regions.min.js

**Reply by anttieintola** [reply]
Thanks! I also found this https://cdnjs.com/libraries/wavesurfer.js/2.0.4 
You approve these? :)

**Reply by katspaugh** [reply]
The last stable version is 6.
